### ISSUE TITLE
cde.error.models in batch

### ISSUE THREAD

**Issue by poryaparvizi** [hms-dbmi/scde#62]
Do you think in scde.error.models you should also take batch effect into consideration?

**Comment by JEFworks** [comment]
Yes, if you have batch information, you can provide that into the groups parameter so that error-modeling is done in a batch-specific manner.

Jean Fan, PhD
Bioinformatics and Integrative Genomics
NCI F99/K00 Post-Doctoral Fellow
Zhuang Lab | Harvard University
12 Oxford St, Naito 031, Cambridge, MA 02138
web: jef.works<http://jefworks.com/>

On Mar 3, 2018, at 8:52 AM, Poorya Parvizi <notifications@github.com<mailto:notifications@github.com>> wrote:


Do you think in scde.error.models you should also take batch effect into consideration?

**Comment by poryaparvizi** [comment]
But the problem is, my single cells have two conditions which both exist in different batches.
